### ISSUE TITLE
Language Server is crashing on startup in Insiders

### ISSUE THREAD

**Issue by gjsjohnmurray** [intersystems/language-server#152]
Issue Type: <b>Bug</b>

1. Have VS Code Insiders installed, plus the InterSystems Language Server extension and whatever it depends on.
2. Start VS Code Insiders.
3. Observe two notifications.
4. Find content like this in its output channel:

```


#
# Fatal error in , line 0
# ignored
#
#
#
#FailureMessage Object: 0000007F1FFFC440
 1: 00007FF73B7112DF node::OnFatalError+70575
 2: 00007FF73C566CFA v8::FunctionTemplate::RemovePrototype+592554
 3: 00007FF73A59E8EA v8::ArrayBuffer::GetBackingStore+922
 4: 00007FF73A59E5AF v8::ArrayBuffer::GetBackingStore+95
 5: 00007FF73B9A8449 napi_get_typedarray_info+425
 6: 00007FFC5C736FC1 
 7: 00007FFC5C73350F 
 8: 00007FFC5C73730E 
 9: 00007FFC5C737467 
10: 00007FFC5C7372B3 
11: 00007FF73B9A9F6A napi_is_detached_arraybuffer+330
12: 00007FF73C017FB1 v8::Object::SetLazyDataProperty+479745
13: 00007FF73C0179FC v8::Object::SetLazyDataProperty+478284
14: 00007FF73C01776B v8::Object::SetLazyDataProperty+477627
15: 00007FF73C017627 v8::Object::SetLazyDataProperty+477303
16: 00007FF73B4EE4FC v8_inspector::protocol::Binary::operator=+455164
17: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
18: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
19: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
20: 00007FF73B48563B v8_inspector::protocol::Binary::operator=+25403
21: 00007FF73B48528C v8_inspector::protocol::Binary::operator=+24460
22: 00007FF73BD60464 Cr_z_deflate+100052
23: 00007FF73BD5E5F2 Cr_z_deflate+92258
24: 00007FF73BD5E05B Cr_z_deflate+90827
25: 00007FF73BD5B367 Cr_z_deflate+79319
26: 00007FF73C0A86F8 v8::Value::IsInt32+345400
27: 00007FF73C0A81E7 v8::Value::IsInt32+344103
28: 00007FF73B4EE3FC v8_inspector::protocol::Binary::operator=+454908
29: 00007FF73B572A56 v8_inspector::protocol::Binary::operator=+997206
30: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
31: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
32: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
33: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
34: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
35: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
36: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
37: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
38: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
39: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
40: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
41: 00007FF73B48563B v8_inspector::protocol::Binary::operator=+25403
42: 00007FF73B48528C v8_inspector::protocol::Binary::operator=+24460
43: 00007FF73BD60464 Cr_z_deflate+100052
44: 00007FF73BD5E5F2 Cr_z_deflate+92258
45: 00007FF73BE1DC7D v8::Function::Call+365
46: 00007FF73B6B2F47 node::CallbackScope::CallbackScope+2423
47: 00007FF73B6B32F9 node::MakeCallback+249
48: 00007FF73B6E7C30 node::EmitAsyncDestroy+198128
49: 00007FF73B7ED9C6 uv_check_stop+150
50: 00007FF73B71B243 uv_run+1267
51: 00007FF73B2EAB11 v8_inspector::protocol::Binary::~Binary+139553
52: 00007FF73B16FE4B GetHandleVerifier+4063963
53: 00007FF73C527F72 v8::FunctionTemplate::RemovePrototype+335138
54: 00007FFC81F97034 BaseThreadInitThunk+20
55: 00007FFC82CA2651 RtlUserThreadStart+33
[Info  - 18:13:00] Connection to server got closed. Server will restart.


#
# Fatal error in , line 0
# ignored
#
#
#
#FailureMessage Object: 000000FBC23FBE90
 1: 00007FF73B7112DF node::OnFatalError+70575
 2: 00007FF73C566CFA v8::FunctionTemplate::RemovePrototype+592554
 3: 00007FF73A59E8EA v8::ArrayBuffer::GetBackingStore+922
 4: 00007FF73A59E5AF v8::ArrayBuffer::GetBackingStore+95
 5: 00007FF73B9A8449 napi_get_typedarray_info+425
 6: 00007FFC5C736FC1 
 7: 00007FFC5C73350F 
 8: 00007FFC5C73730E 
 9: 00007FFC5C737467 
10: 00007FFC5C7372B3 
11: 00007FF73B9A9F6A napi_is_detached_arraybuffer+330
12: 00007FF73C017FB1 v8::Object::SetLazyDataProperty+479745
13: 00007FF73C0179FC v8::Object::SetLazyDataProperty+478284
14: 00007FF73C01776B v8::Object::SetLazyDataProperty+477627
15: 00007FF73C017627 v8::Object::SetLazyDataProperty+477303
16: 00007FF73B4EE4FC v8_inspector::protocol::Binary::operator=+455164
17: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
18: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
19: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
20: 00007FF73B48563B v8_inspector::protocol::Binary::operator=+25403
21: 00007FF73B48528C v8_inspector::protocol::Binary::operator=+24460
22: 00007FF73BD60464 Cr_z_deflate+100052
23: 00007FF73BD5E5F2 Cr_z_deflate+92258
24: 00007FF73BD5E05B Cr_z_deflate+90827
25: 00007FF73BD5B367 Cr_z_deflate+79319
26: 00007FF73C0A86F8 v8::Value::IsInt32+345400
27: 00007FF73C0A81E7 v8::Value::IsInt32+344103
28: 00007FF73B4EE3FC v8_inspector::protocol::Binary::operator=+454908
29: 00007FF73B572A56 v8_inspector::protocol::Binary::operator=+997206
30: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
31: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
32: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
33: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
34: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
35: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
36: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
37: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
38: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
39: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
40: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
41: 00007FF73B48563B v8_inspector::protocol::Binary::operator=+25403
42: 00007FF73B48528C v8_inspector::protocol::Binary::operator=+24460
43: 00007FF73BD60464 Cr_z_deflate+100052
44: 00007FF73BD5E5F2 Cr_z_deflate+92258
45: 00007FF73BE1DC7D v8::Function::Call+365
46: 00007FF73B6B2F47 node::CallbackScope::CallbackScope+2423
47: 00007FF73B6B32F9 node::MakeCallback+249
48: 00007FF73B6E7C30 node::EmitAsyncDestroy+198128
49: 00007FF73B7ED9C6 uv_check_stop+150
50: 00007FF73B71B243 uv_run+1267
51: 00007FF73B2EAB11 v8_inspector::protocol::Binary::~Binary+139553
52: 00007FF73B16FE4B GetHandleVerifier+4063963
53: 00007FF73C527F72 v8::FunctionTemplate::RemovePrototype+335138
54: 00007FFC81F97034 BaseThreadInitThunk+20
55: 00007FFC82CA2651 RtlUserThreadStart+33
[Info  - 18:13:02] Connection to server got closed. Server will restart.


#
# Fatal error in , line 0
# ignored
#
#
#
#FailureMessage Object: 000000224DBFC1E0
 1: 00007FF73B7112DF node::OnFatalError+70575
 2: 00007FF73C566CFA v8::FunctionTemplate::RemovePrototype+592554
 3: 00007FF73A59E8EA v8::ArrayBuffer::GetBackingStore+922
 4: 00007FF73A59E5AF v8::ArrayBuffer::GetBackingStore+95
 5: 00007FF73B9A8449 napi_get_typedarray_info+425
 6: 00007FFC5C736FC1 
 7: 00007FFC5C73350F 
 8: 00007FFC5C73730E 
 9: 00007FFC5C737467 
10: 00007FFC5C7372B3 
11: 00007FF73B9A9F6A napi_is_detached_arraybuffer+330
12: 00007FF73C017FB1 v8::Object::SetLazyDataProperty+479745
13: 00007FF73C0179FC v8::Object::SetLazyDataProperty+478284
14: 00007FF73C01776B v8::Object::SetLazyDataProperty+477627
15: 00007FF73C017627 v8::Object::SetLazyDataProperty+477303
16: 00007FF73B4EE4FC v8_inspector::protocol::Binary::operator=+455164
17: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
18: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
19: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
20: 00007FF73B48563B v8_inspector::protocol::Binary::operator=+25403
21: 00007FF73B48528C v8_inspector::protocol::Binary::operator=+24460
22: 00007FF73BD60464 Cr_z_deflate+100052
23: 00007FF73BD5E5F2 Cr_z_deflate+92258
24: 00007FF73BD5E05B Cr_z_deflate+90827
25: 00007FF73BD5B367 Cr_z_deflate+79319
26: 00007FF73C0A86F8 v8::Value::IsInt32+345400
27: 00007FF73C0A81E7 v8::Value::IsInt32+344103
28: 00007FF73B4EE3FC v8_inspector::protocol::Binary::operator=+454908
29: 00007FF73B572A56 v8_inspector::protocol::Binary::operator=+997206
30: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
31: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
32: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
33: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
34: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
35: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
36: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
37: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
38: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
39: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
40: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
41: 00007FF73B48563B v8_inspector::protocol::Binary::operator=+25403
42: 00007FF73B48528C v8_inspector::protocol::Binary::operator=+24460
43: 00007FF73BD60464 Cr_z_deflate+100052
44: 00007FF73BD5E5F2 Cr_z_deflate+92258
45: 00007FF73BE1DC7D v8::Function::Call+365
46: 00007FF73B6B2F47 node::CallbackScope::CallbackScope+2423
47: 00007FF73B6B32F9 node::MakeCallback+249
48: 00007FF73B6E7C30 node::EmitAsyncDestroy+198128
49: 00007FF73B7ED9C6 uv_check_stop+150
50: 00007FF73B71B243 uv_run+1267
51: 00007FF73B2EAB11 v8_inspector::protocol::Binary::~Binary+139553
52: 00007FF73B16FE4B GetHandleVerifier+4063963
53: 00007FF73C527F72 v8::FunctionTemplate::RemovePrototype+335138
54: 00007FFC81F97034 BaseThreadInitThunk+20
55: 00007FFC82CA2651 RtlUserThreadStart+33
[Info  - 18:13:03] Connection to server got closed. Server will restart.


#
# Fatal error in , line 0
# ignored
#
#
#
#FailureMessage Object: 0000001D60DFBE40
 1: 00007FF73B7112DF node::OnFatalError+70575
 2: 00007FF73C566CFA v8::FunctionTemplate::RemovePrototype+592554
 3: 00007FF73A59E8EA v8::ArrayBuffer::GetBackingStore+922
 4: 00007FF73A59E5AF v8::ArrayBuffer::GetBackingStore+95
 5: 00007FF73B9A8449 napi_get_typedarray_info+425
 6: 00007FFC5C736FC1 
 7: 00007FFC5C73350F 
 8: 00007FFC5C73730E 
 9: 00007FFC5C737467 
10: 00007FFC5C7372B3 
11: 00007FF73B9A9F6A napi_is_detached_arraybuffer+330
12: 00007FF73C017FB1 v8::Object::SetLazyDataProperty+479745
13: 00007FF73C0179FC v8::Object::SetLazyDataProperty+478284
14: 00007FF73C01776B v8::Object::SetLazyDataProperty+477627
15: 00007FF73C017627 v8::Object::SetLazyDataProperty+477303
16: 00007FF73B4EE4FC v8_inspector::protocol::Binary::operator=+455164
17: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
18: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
19: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
20: 00007FF73B48563B v8_inspector::protocol::Binary::operator=+25403
21: 00007FF73B48528C v8_inspector::protocol::Binary::operator=+24460
22: 00007FF73BD60464 Cr_z_deflate+100052
23: 00007FF73BD5E5F2 Cr_z_deflate+92258
24: 00007FF73BD5E05B Cr_z_deflate+90827
25: 00007FF73BD5B367 Cr_z_deflate+79319
26: 00007FF73C0A86F8 v8::Value::IsInt32+345400
27: 00007FF73C0A81E7 v8::Value::IsInt32+344103
28: 00007FF73B4EE3FC v8_inspector::protocol::Binary::operator=+454908
29: 00007FF73B572A56 v8_inspector::protocol::Binary::operator=+997206
30: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
31: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
32: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
33: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
34: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
35: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
36: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
37: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
38: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
39: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
40: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
41: 00007FF73B48563B v8_inspector::protocol::Binary::operator=+25403
42: 00007FF73B48528C v8_inspector::protocol::Binary::operator=+24460
43: 00007FF73BD60464 Cr_z_deflate+100052
44: 00007FF73BD5E5F2 Cr_z_deflate+92258
45: 00007FF73BE1DC7D v8::Function::Call+365
46: 00007FF73B6B2F47 node::CallbackScope::CallbackScope+2423
47: 00007FF73B6B32F9 node::MakeCallback+249
48: 00007FF73B6E7C30 node::EmitAsyncDestroy+198128
49: 00007FF73B7ED9C6 uv_check_stop+150
50: 00007FF73B71B243 uv_run+1267
51: 00007FF73B2EAB11 v8_inspector::protocol::Binary::~Binary+139553
52: 00007FF73B16FE4B GetHandleVerifier+4063963
53: 00007FF73C527F72 v8::FunctionTemplate::RemovePrototype+335138
54: 00007FFC81F97034 BaseThreadInitThunk+20
55: 00007FFC82CA2651 RtlUserThreadStart+33
[Info  - 18:13:04] Connection to server got closed. Server will restart.


#
# Fatal error in , line 0
# ignored
#
#
#
#FailureMessage Object: 00000064AABFC0A0
 1: 00007FF73B7112DF node::OnFatalError+70575
 2: 00007FF73C566CFA v8::FunctionTemplate::RemovePrototype+592554
 3: 00007FF73A59E8EA v8::ArrayBuffer::GetBackingStore+922
 4: 00007FF73A59E5AF v8::ArrayBuffer::GetBackingStore+95
 5: 00007FF73B9A8449 napi_get_typedarray_info+425
 6: 00007FFC5C736FC1 
 7: 00007FFC5C73350F 
 8: 00007FFC5C73730E 
 9: 00007FFC5C737467 
10: 00007FFC5C7372B3 
11: 00007FF73B9A9F6A napi_is_detached_arraybuffer+330
12: 00007FF73C017FB1 v8::Object::SetLazyDataProperty+479745
13: 00007FF73C0179FC v8::Object::SetLazyDataProperty+478284
14: 00007FF73C01776B v8::Object::SetLazyDataProperty+477627
15: 00007FF73C017627 v8::Object::SetLazyDataProperty+477303
16: 00007FF73B4EE4FC v8_inspector::protocol::Binary::operator=+455164
17: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
18: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
19: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
20: 00007FF73B48563B v8_inspector::protocol::Binary::operator=+25403
21: 00007FF73B48528C v8_inspector::protocol::Binary::operator=+24460
22: 00007FF73BD60464 Cr_z_deflate+100052
23: 00007FF73BD5E5F2 Cr_z_deflate+92258
24: 00007FF73BD5E05B Cr_z_deflate+90827
25: 00007FF73BD5B367 Cr_z_deflate+79319
26: 00007FF73C0A86F8 v8::Value::IsInt32+345400
27: 00007FF73C0A81E7 v8::Value::IsInt32+344103
28: 00007FF73B4EE3FC v8_inspector::protocol::Binary::operator=+454908
29: 00007FF73B572A56 v8_inspector::protocol::Binary::operator=+997206
30: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
31: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
32: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
33: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
34: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
35: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
36: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
37: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
38: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
39: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
40: 00007FF73B4879EF v8_inspector::protocol::Binary::operator=+34543
41: 00007FF73B48563B v8_inspector::protocol::Binary::operator=+25403
42: 00007FF73B48528C v8_inspector::protocol::Binary::operator=+24460
43: 00007FF73BD60464 Cr_z_deflate+100052
44: 00007FF73BD5E5F2 Cr_z_deflate+92258
45: 00007FF73BE1DC7D v8::Function::Call+365
46: 00007FF73B6B2F47 node::CallbackScope::CallbackScope+2423
47: 00007FF73B6B32F9 node::MakeCallback+249
48: 00007FF73B6E7C30 node::EmitAsyncDestroy+198128
49: 00007FF73B7ED9C6 uv_check_stop+150
50: 00007FF73B71B243 uv_run+1267
51: 00007FF73B2EAB11 v8_inspector::protocol::Binary::~Binary+139553
52: 00007FF73B16FE4B GetHandleVerifier+4063963
53: 00007FF73C527F72 v8::FunctionTemplate::RemovePrototype+335138
54: 00007FFC81F97034 BaseThreadInitThunk+20
55: 00007FFC82CA2651 RtlUserThreadStart+33
[Error - 18:13:05] Connection to server got closed. Server will not be restarted.
```

Extension version: 1.1.4
VS Code version: Code - Insiders 1.56.0-insider (7ce1499d9541ba2acf66cd30a9054bd7818a8f80, 2021-04-22T05:13:14.711Z)
OS version: Windows_NT x64 10.0.19042

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-10510U CPU @ 1.80GHz (8 x 2304)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.84GB (0.56GB free)|
|Process Argv|-n --enable-proposed-api intersystems-community.vscode-objectscript --crash-reporter-id 584c5b29-25dd-4f1c-a7c6-fb3b6998cbc6|
|Screen Reader|no|
|VM|0%|
</details>

**Comment by isc-bsaviano** [comment]
Andy needs to look at this. The errors are coming from the modules he uses to call out to the parsers.

**Comment by gjsjohnmurray** [comment]
Verified fixed in 1.1.5-beta.2
